modules/ossm-supported-configurations.adoc

```diff
@@ -32,7 +32,7 @@ For additional information about {ProductName} lifecycle and supported configura
 {ProductName} supports the following network configurations.
 
 * OpenShift-SDN
-* OVN-Kubernetes is supported as a technology preview in {product-title} version 4.7 and newer versions.
+* OVN-Kubernetes is supported on OpenShift Container Platform 4.7.32+, OpenShift Container Platform 4.8.12+, and OpenShift Container Platform 4.9+.
```

Comment:
```suggestion
* OVN-Kubernetes is supported on {product-title} 4.7.32+, {product-title} 4.8.12+, and {product-title} 4.9+.
```
Use variable instead of hardcoding it in :)